Server/src/tests/services/DataSetService.spec.ts

```diff
@@ -51,7 +51,7 @@ describe('data service test', () => {
     let testData: IDataRequestModel;
     testData = {} as any;
     testData.datasetId = undefined;
-    testData.material = ["carbon, graphite, pressed graphite"];
+    testData.material = ["O2", "carbon, graphite, pressed graphite"];

```

Comment:
why the change?

Reply:
Its a test. I think it was just meant to verify

Reply:
It originally had O2 and was removed in the search overhaul during some debugging. I realized after the PR was closed that this was not restored and thus this is going back to how it should be, versus a new change.

Reply:
Sounds good